.ado/windows-vs-pr.yml

```diff
@@ -315,6 +315,49 @@ jobs:
           #rerunFailedTestCasesMaxLimit: '5' # Optional
           #rerunMaxAttempts: '3' # Optional
 
+      - task: VSTest@2
+        displayName: Run Desktop Unit Tests (Google Test)
+        timeoutInMinutes: 5 # Set smaller timeout for UTs, since there have been some hangs, and this allows the job to timeout quicker
+        inputs:
+          testSelector: 'testAssemblies' # Options: testAssemblies, testPlan, testRun
+          testAssemblyVer2: 'JSI.Desktop.UnitTests\JSI.Desktop.UnitTests.exe'  # Required when testSelector == TestAssemblies
+          #testPlan: # Required when testSelector == TestPlan
+          #testSuite: # Required when testSelector == TestPlan
+          #testConfiguration: # Required when testSelector == TestPlan
+          #tcmTestRun: '$(test.RunId)' # Optional
```

Comment:
if these are commented out, remove

Reply:
Not sure a new task is needed. Please check if `testAssemblies` accepts multiple binaries. If so, the existing Unit Tests task could be used for this EXE too.

Reply:
Done.

Reply:
Judging from sdx-platform\.ado\windows-pr.yml, `testAssemblies` does accept multiple binaries, but when I tried it here, I go this warning:
`##[warning]No test assemblies found matching the pattern: 'React.Windows.Desktop.UnitTests\React.Windows.Desktop.UnitTests.dll','JSI.Desktop.UnitTests\JSI.Desktop.UnitTests.exe'.` I am not sure why.

---
In reply to: [298772927](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r298772927) [](ancestors = 298772927)